docs/learning/ECSqlReference/ECSqlFunctions.md

```diff
@@ -0,0 +1,188 @@
+# ECSQL Built-In functions
+
+Following is list of built-in scalar functions
+
+ECSQL allows use of these built-in functions:
+
+1. [`ec_classname()`](#ec_classname-ecclassid--format-string--format-id) - Gets the formatted/qualified class name, given ECClassId as input
+1. [`ec_classid())`](#ec_classidschema-name-or-alias----classname) - Gets ECClassId, given a formatted/qualified class name as input
+1. [`regexp()`](#regexp--regex-value) - test if a text matches a regex.
+1. [`regexp_extract()`](#regexp_extract--value-regex--rewrite) - extract and rewrite matching regex group from a string value.
+1. [`strToGuid()`](#strtoguid-guid-string) - covert string guid to binary guid.
+1. [`guidToStr()`](#guidtostring-binary-guid) - covert binary guid to string guid.
+1. [`navigation_value()`](#navigation_value-ecnavigationproperty-path-id--RelECClassId) - Constructs an ECNavigation value, given ECNavigation property, Id and optionaly, RelECClassId.
+
+## ec_classname( _ecclassId_ [, _format-string_ | _format-id_] )
+
+For the specified ecClassId, returns the class name as a string formatted according to the specified format-string
+
+### Parameters
+
+`ecclassId`: An integer which could be a constant, column or a parameter.
+`format-string | format-id`: Optional format specifier and could be one of the following values. `NULL` is also valid value -- this is the same as not specifying the second parameter at all
```

Comment:
`this is the same as not specifying the second parameter at all`

As if the second parameter were not specified? Not sure what the intent is...